includes/helpers/functions/templates-ui.php

```diff
@@ -329,7 +329,7 @@ function wpbdp_main_links( $buttons = null ) {
 
 	if ( in_array( 'create', $buttons, true ) ) {
 		$html[] = '<a href="' . esc_url( wpbdp_url( 'submit_listing' ) ) . '" id="wpbdp-bar-submit-listing-button" class="button wpbdp-button">' .
-			esc_html__( 'Add Listing', 'business-directory-plugin' ) .
+			esc_html__( '+ Add Listing', 'business-directory-plugin' ) .
```

Comment:
Hi Steph, thanks for the review.

Actually, these are basic styles and we'll focus on specific parts in the component PRs. For instance, in this PR, we only set basic button styles to avoid repetition.